docs/api/cozy-stack-client.md

```diff
@@ -1256,9 +1272,9 @@ Fetches all OAuth clients
 | Param | Type | Description |
 | --- | --- | --- |
 | options | <code>object</code> | Query options |
-| options.limit | <code>number</code> | For pagination, the number of results to return. |
-| options.bookmark | <code>object</code> | For cursor-based pagination, the index cursor. |
-| options.keys | <code>Array</code> | Ids of specific clients to return (within the current page), |
+| [options.limit] | <code>number</code> | For pagination, the number of results to return. |
+| [options.bookmark] | <code>string</code> | For bookmark-based pagination, the document _id to start from |
+| [options.keys] | <code>Array.&lt;string&gt;</code> | Ids of specific clients to return (within the current page), |
```

Comment:
Array.&lt;string&gt; ???

Reply:
I don't know why a dot was added between `Array` and `<string>` but the type definition in `OAuthClientsCollection` is `Array<string>`.
This doc being generated, I think we should rather review the type definition.